### DISCUSSION TITLE
Welcome to Poddr Discussions!

### DISCUSSION THREAD

**Discussion by Sn8z** [Sn8z/Poddr#813] [General]
## 👋 Welcome!
  Poddr is using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by rrinjapan** [comment]
Love the App! Thank you!
